### DISCUSSION TITLE
Relay trigger by upper and lower alarm

### DISCUSSION THREAD

**Discussion by InsertAnyName** [theyosh/TerrariumPI#857] [Q&A] [answered]
First of all, thank you to everyone who contributed to the software.
After some difficulties I'm using TerrariumPi 4.7 on a Pi 3b, my operating system is Raspberry Pi OS Lite V11 (Bullseye) as the installation failed repeatedly on V12.
I've managed to get all the sensors and relays I need to work.
My Question: I would like to dehumidify my housing. My plan is to turn on an outlet with a dehumidifier when the humidity is above the high alarm value and turn it off when it falls below the low alarm value. Is that possible, and if so, how? Unfortunately, I haven't had any success with my search so far.
Thanks!

**Comment by theyosh** [comment, marked as answer]
This can be done.

First make sure you have setup the humidity sensor with the correct alarm and limit values. I would start with limit values of 0 and 100. Than set the alarm values to the values you want to have in your house. The max alarm value will be used to trigger the relay. So lets say you want no more than 60% humidity, set the alarm max value to 60.

You should create an enclosure. Called it 'home' as it is your home you want to control
Then create an area with `humidity` type
Select `sensor` mode
Select the sensor(s) you want to use

Then select the `high alarm` tab and select the relay for your dehumidifier.

Save the form.

This should do it. The outlet goes on when the high alarm is reached, and shutdown when the low alarm value is reached.

**Reply by InsertAnyName** [reply]
Thanks. It worked. I thought that I hat to select an additional trigger (low alarm) for the relay to be switched off. Sometimes things are simple ;)
